src/Redux/api.tsx

```diff
@@ -56,6 +57,7 @@ const routes = {
   forgotPassword: {
     path: "/api/v1/password_reset/",
     method: "POST",
+    TRes: Res<IConfig>(),
```

Comment:
Does the `password_reset` endpoint give a response of `IConfig`?

Feel free to create new interface definitions if the proper interface doesn't exist in the codebase.

Reply:
How do  i test the response? to get the type

Reply:
<img width="1131" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/4821c88b-36cd-4cc2-b342-96afba99f869">